docs/configuration/config-file-reference.md

```diff
@@ -1426,6 +1426,42 @@ storage:
 # CLI flag: -ruler.notification-timeout
 [notification_timeout: <duration> | default = 10s]
 
+alertmanager_client:
+  tls:
```

Comment:
What do you think to inline it? In the gRPC client we've inlined it, and the YAML structure doesn't match the CLI flag (flag is `-ruler.alertmanager-client.tls-cert-path` while to match YAML it would be `-ruler.alertmanager-client.tls.tls-cert-path` but the extra `tls.` would be a repetition).

Reply:
👍